docs/README.adoc

```diff
@@ -1,31 +1,27 @@
 = Aktualizr reference docs
 
-This directory contains developer-focused documentation on various specialized aktualizr topics.
+This directory contains developer-focused documentation on various specialized aktualizr topics. Some of these topics are currently hosted in a separate documentation repository.
 
-link:./automatic-provisioning.adoc[automatic-provisioning.adoc] - Describes how aktualizr handles automatic provisioning.
+link:./client-provisioning-methods.adoc[client-provisioning-methods.adoc] - Describes how aktualizr handles provisioning with either shared credentials or device credentials. 
 
 link:./configuration.adoc[configuration.adoc] - A reference for aktualizr's TOML config files.
 
-link:./credentials.adoc[credentials.adoc] - An explanation of the format of credentials.zip, a bundle generated by HERE OTA Connect/OTA Community Edition containing necessary credentials and config for aktualizr and meta-updater.
-
-link:./deb-package-install.adoc[deb-package-install.adoc] - The https://github.com/advancedtelematic/aktualizr/releases[releases] page includes a .deb package for aktualizr. This isn't likely useful for most production use cases, but it can be handy for testing. This document guides you through how to use the .deb.
+link:./provisioning-methods-and-credentialszip.adoc[provisioning-methods-and-credentialszip.adoc] - An explanation of the format of credentials.zip, a bundle generated by HERE OTA Connect/OTA Community Edition containing necessary credentials and config for aktualizr and meta-updater.
```

Comment:
In docs, you generally organize information by reader needs not by alphabet (unless you're creating an index). Otherwise, a topic called "Introduction" would be in the middle of the list which is pretty weird. I'll leave this as-is for now, and add a more structured TOC (which I think what this is trying to be) in the next PR.

Reply:
Not a problem. Originally this doc was just an overview of the items in the docs folder, so I think the alphabetization was just for convenience. There wasn't really any intended structure to it, to my knowledge at least. Since that has changed, or seems like it will change, feel free to put it in whatever order you want, as long as there is some logic to it.